redpanda.yaml

```diff
@@ -0,0 +1,151 @@
+package:
+  name: redpanda
+  version: "25.1.4"
+  description: "Redpanda is a streaming platform based on Apache Kafka API"
+  resources:
+    cpu: 16
+    memory: 16Gi
```

Comment:
go/wolfi-rsc/redpanda

I'd give it about 4x the CPU: 
<img width="442" alt="image" src="https://github.com/user-attachments/assets/0d6761f4-e1be-4c60-87fc-b324602ef883" />